### PR TITLE
PWN-6677 - single line recycler for ActionButton

### PR DESCRIPTION
**Pull request by NikolaiEmelianov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6677

## Description of Work

Single line recycler for ActionButton

### CODE REVIEW

**Comment by gslevinkov** [comment]
hi, Nikolai. our commit writing formatting rule is:
`PWN-6677 - single line recycler for ActionButton`
instead of 
`PWN-6677 Single line recycler for ActionButton` (we put dash and start message from lowercase)

File: app/src/main/res/layout/layout_action_buttons.xml [hunk 1]
```diff
@@ -9,10 +9,12 @@
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/recyclerViewActionButtons"
-        android:layout_width="match_parent"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_gravity="center"
         android:orientation="horizontal"
-        app:layoutManager="com.google.android.flexbox.FlexboxLayoutManager"
+        android:overScrollMode="never"
```
Comment (gslevinkov) [inline app/src/main/res/layout/layout_action_buttons.xml:16]: 🤯 how is this possible??

Comment (NikolaiEmelianov) [reply app/src/main/res/layout/layout_action_buttons.xml:16]: what is you mean? short list, overScroll not need

Comment (gslevinkov) [reply app/src/main/res/layout/layout_action_buttons.xml:16]: i meant that you replaced FlexboxLayoutManager with LinearLayoutManager and now it works okay? can you send the screenshot of how the buttons look like now?

Comment (NikolaiEmelianov) [reply app/src/main/res/layout/layout_action_buttons.xml:16]: photo_2023-01-06 12 00 39
photo_2023-01-06 12 00 33
photo_2023-01-06 12 00 30

**Comment by gslevinkov** [comment]
i checked it myself, looks good

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**